edm4eic.yaml

```diff
@@ -372,39 +391,40 @@ datatypes:
-      - float             time              // Track time [ns]    
-      - float             timeError         // Error on the time
-      - float             charge            // Particle charge
+      - int32_t              type              // Type of track parameters (-1/seed, 0/head, ...)
+      - uint64_t             surface           // Surface for bound parameters (geometryID)
+      - edm4hep::Vector2f    loc               // 2D location on surface
+      - float                theta             // Track polar angle [rad]
+      - float                phi               // Track azimuthal angle [rad]
+      - float                qOverP            // [e/GeV]
+      - float                time              // Track time [ns] 
+      - edm4eic::Cov6f       covariance        // Full covariance in basis [l0,l1,theta,phi,q/p,t]
+  
     OneToOneRelations:
       - edm4eic::Trajectory trajectory      // Trajectory associated with these track parameters
 
+
   edm4eic::Track:
     Description: "Track information at the vertex"
-    Author: "S. Joosten"
+    Author: "S. Joosten, J. Osborn"
     Members:
-      - int32_t           type              // Flag that defines the type of track
-      - float             chi2              // Total chi2 (sum) of the track fit
-      - int32_t           ndf               // Numbers of degrees of freedom of the track fit
-      - edm4hep::Vector3f momentum          // Track 3-momentum at the vertex [GeV]
-      - edm4eic::Cov3f    momentumError     // Covariance matrix on the momentum
-      - float             time              // Track time at the vertex [ns]
-      - float             timeError         // Error on the track vertex time
-      - float             charge            // Particle charge
+      - int32_t            type                           // Flag that defines the type of track
+      - edm4hep::Vector3f  position                       // Track 3-position at the vertex 
+      - edm4hep::Vector3f  momentum                       // Track 3-momentum at the vertex [GeV]
+      - edm4eic::Cov6f     positionMomentumCovariance     // Covariance matrix in basis [x,y,z,px,py,pz]
+      - float              time                           // Track time at the vertex [ns]
+      - float              timeError                      // Error on the track vertex time
+      - float              charge                         // Particle charge
+      - float              chi2                           // Total chi2
+      - uint32_t           ndf                            // Number of degrees of freedom
+      - uint32_t           pid                            // PDG particle ID hypothesis
     OneToOneRelations:
-      - edm4eic::Trajectory trajectory      // Trajectory of this track
-      - edm4eic::Vertex   vertex            // Track vertex of this track
+      - edm4eic::Trajectory                     trajectory      // Trajectory of this track
```

Comment:
Are you still keeping `Trajectory`? I thought we were going straight from TrackParameters to Track now? I guess I'm a little unclear on how you see the data flow?

Reply:
A track stores a set of track parameters and a trajectory. The track parameters store the track state information at all of the surfaces, the trajectory stores global track quantities like the number of measurements used in the fit, and the Track itself stores the track parameters at the vertex. In principle we could dump the Trajectory entirely and just store this information in the Track itself since they are completely one-to-one, I thought that the distinction was useful though because the Trajectory is really tracking expert information while the Track (that holds the track parameters at the vertex) is really analysis level information.

Reply:
OK, I think I agree. It can become cumbersome if we always have to deal with relations, but if it's expert only it makes sense to separate them (and then have the option of only storing Tracks)